### ISSUE TITLE
Question about Press Time slot

### ISSUE THREAD

**Issue by george-alisson** [Ryochan7/antimicro#183]
Should not the Press Time slot just set the minimum press time as the specified time but not release the key? Because that is what happens.

Of course that is your design choice, but I think it would make more sense that if the button is still pressed, the key is not released.

**Comment by Ryochan7** [comment]
That was a carry over from Xpadder. It is mainly useful for release zones so there is the assumption that the key should be released after a set period of time. There are times when keys in a sequence should only be pressed for a period of time while a button is still held. If that new behavior were implemented, it would probably be added as a different slot type.

**Comment by george-alisson** [comment]
Mm, That is ok. I don't have a game that need this, but I could see the use of it. I asked this because the Repeat slot that I implemented could use this press time too, if the key would not be released.

BTW I want to thank you for this program. I used to use JoyToKey and Xpadder for some time, but now this I found this I'm using it and I don't think I'll ever need to go back. antimicro have more features, is free, and better yet, is open source, meaning that if there is something I'd like it to have, I cloud implement it myself. So, thanks!
